cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestBinaryTimeSeriesEndpointInput.java

```diff
@@ -50,11 +50,13 @@ void testGetOneQueryRequest() {
         Instant end = ZonedDateTime.of(2018, 2, 5, 0, 0, 0, 0, ZoneId.of("UTC")).toInstant();
         BinaryTimeSeriesEndpointInput.GetAll input = BinaryTimeSeriesEndpointInput.getAll("arbu.Binary.Inst.1Hour.0.Ccp-Rev", "SWT", start, end)
                 .pageSize(10)
+                .versionDate(start)
```

Comment:
I have been bit before getting parameters accidentally mixed up, so I've been trying to make each parameter unique to ensure they aren't crisscrossed. I recommend creating a new date (applies to the other tests too)

Reply:
Added new date object for version date to test against